packages/fxa-auth-server/lib/account-events.ts

```diff
@@ -64,19 +64,11 @@ export class AccountEventsManager {
 
       // Firestore can be configured to ignore undefined keys, but we do it here
       // since it is a global config
-      const filteredEmailEvent = {};
-      Object.keys(emailEvent).forEach((key) => {
-        // @ts-ignore
-        if (emailEvent[key]) {
-          // @ts-ignore
-          filteredEmailEvent[key] = emailEvent[key];
-        }
-      });
+      for (const [key, value] of Object.entries(emailEvent)) {
+        if (!value) delete emailEvent[key as keyof typeof emailEvent];
```

Comment:
Kind of surprised this wouldn't have violated some ESLint error like [`curly`](https://eslint.org/docs/latest/rules/curly). Unless maybe we're setting it in eslint-plugin-fxa and it's being disabled by prettier config. 

https://github.com/mozilla/eslint-plugin-fxa/blob/fbcfb9db9881d644d147fa9461a4273884160553/lib/configs/default.js#L21